app/api/tweets/[id]/lastComment/route.ts

```diff
@@ -0,0 +1,17 @@
+import {TweetParams} from "@/app/api/tweets/[id]/definitions";
+import {ErrorHandler} from "@/app/lib/util/ErrorHandler";
+import {getLatestComment} from "@/app/lib/comment/commentDbManager";
+import {Reply} from "@/app/lib/api/Reply";
+
+export async function GET(request: Request, {params: {id}}: TweetParams) {
+    try {
+        const comment = await getLatestComment(id)
+        if (!comment) {
+            return Reply.send({comment: null})
```

Comment:
Since you're using findFirst, instead of returning a response with `null`, you should instead **throw** an `AppError` with a **404** status code, and let the `ErrorHandler` do the rest. Check what has been implemented and use those features, they are there for a reason!